gcloud-java-storage/src/main/java/com/google/gcloud/spi/DefaultStorageRpc.java

```diff
@@ -163,8 +166,20 @@ public StorageObject create(StorageObject storageObject, final InputStream conte
           .setPageToken(PAGE_TOKEN.getString(options))
           .setFields(FIELDS.getString(options))
           .execute();
-      return Tuple.<String, Iterable<StorageObject>>of(
-          objects.getNextPageToken(), objects.getItems());
+      Iterable<StorageObject> storageObjects = Iterables.concat(
+          objects.getItems() != null ? objects.getItems() : ImmutableList.<StorageObject>of(),
```

Comment:
[`MoreObjects.firstNonNull`](http://docs.guava-libraries.googlecode.com/git/javadoc/com/google/common/base/MoreObjects.html#firstNonNull(T,%20T)) could make it slightly nicer.